src/lib.rs

```diff
@@ -113,13 +115,25 @@ pub use miniscript::Miniscript;
 pub trait MiniscriptKey:
     Clone + Eq + Ord + str::FromStr + fmt::Debug + fmt::Display + hash::Hash
 {
+    /// Check if the publicKey is uncompressed. The default
+    /// implementation returns false for all Generic types of
+    /// MiniscriptKey, and returns the true only for
+    /// bitcoin::Publickey if the underlying key is uncompressed.
```

Comment:
This comment could be shortened to just "The default implementation returns false"